_posts/2020-07-31-kubeflow-1.1-blog-post.md

```diff
@@ -0,0 +1,39 @@
+**Kubeflow 1.1 improves ML Workflow Productivity, Isolation & Security, and GitOps**
```

Comment:
Can you add front matter to your blog post? https://github.com/fastai/fastpages#customizing-blog-posts-with-front-matter

```yaml
---
title: "My Title"
description: "Awesome description"
layout: post
toc: false
comments: true
image: images/some_folder/your_image.png
hide: false
search_exclude: true
categories: [fastpages, jupyter]
metadata_key1: metadata_value1
metadata_key2: metadata_value2
---
```